test/test_encryption.py

```diff
@@ -2099,5 +2099,92 @@ def test_05_roundtrip_encrypted_unindexed(self):
         self.assertEqual(decrypted, val)
 
 
+class TestQueryableEncryptionDocsExample(EncryptionIntegrationTest):
+    # Queryable Encryption is not supported on Standalone topology.
+    @client_context.require_no_standalone
+    @client_context.require_version_min(6, 0, -1)
+    def setUp(self):
+        super().setUp()
+
+    def test_queryable_encryption(self):
+        # Drop data from prior test runs.
+        MongoClient().keyvault.datakeys.drop()
+        MongoClient().docs_examples.encrypted.drop()
+
+        kms_providers_map = {"local": {"key": LOCAL_MASTER_KEY}}
+
+        # Create two data keys.
+        key_vault_client = MongoClient()
+        client_encryption = ClientEncryption(
+            kms_providers_map, "keyvault.datakeys", key_vault_client, CodecOptions()
+        )
+        key1_id = client_encryption.create_data_key("local")
+        key2_id = client_encryption.create_data_key("local")
+        key_vault_client.close()
+
+        # Create an encryptedFieldsMap.
```

Comment:
I thought we needed to create the collection (or collMod) with encryptedFields before being able to write/query. Is that not the case?

Reply:
I don't follow the question, we aren't calling `insert_one` until we have a client wit the `encrypted_fields_map`.

Reply:
In queryable encryption the app is expected to create the collection with encryptedFields:
```
db.createCollection('coll', encrypted_fields={...})
```

Reply:
That's only if the collection isn't called out in the `encrypted_fields_map` in `AutoEncryption`.

Reply:
ex: https://github.com/mongodb/mongo-go-driver/pull/969/files#diff-35d11cdcb894289a5379a0fb50235f1933e872f4f591cfd2583245f43f4ee8fcR545

Reply:
Right, the Go example correctly calls create_collection, while this test is missing create_collection. It still needs to be added:
```python
encrypted_coll = encrypted_client.docs_examples.create_collection("encrypted")
```

Reply:
Gotcha, done.